packages/cxl-lumo-styles/src/utils.js

```diff
@@ -1,5 +1,5 @@
 // @see https://github.com/vaadin/vaadin-themable-mixin/blob/v1.5.2/register-styles.html
```

Comment:
This reference URL has become invalid now.

https://github.com/vaadin/web-components/blob/23.1/packages/vaadin-themable-mixin/vaadin-themable-mixin.js#L36-L58 has received a significant refactor.

Not even sure if our util also needs further help, or what exactly (separate PR).

Maybe something like

```suggestion
/**
 * @see https://github.com/vaadin/web-components/blob/23.1/packages/vaadin-themable-mixin/vaadin-themable-mixin.js#L36-L58
 * @todo Align with Vaadin v23.1+.
 */

Reply:
LGTM

Reply:
I found clarity on this legacy adapter, it does only 2 things

* we are able to retire `<custom-style>` elements in https://github.com/conversionxl/aybolit/pull/237
* `<template>` usage is a larger piece, but we can just import `template-renderer.js` now